docs/build/schema.md

```diff
@@ -91,7 +91,7 @@ Key | Required | Type | Description
 
 ## /AuthenticationBasicConfigSchema
 
-Config for basic authentication schema.
+Config for Basic Authentication. Leave empty if your app uses Basic Auth.
```

Comment:
Might be a bit confusing if all of these say "leave empty" without an explanation of why.